pkg/certificate/store/store_windows.go

```diff
@@ -0,0 +1,154 @@
+package store
+
+import (
+	"crypto/x509"
+	"fmt"
+	"syscall"
+	"unsafe"
+)
+
+const (
+	CRYPT_E_NOT_FOUND = 0x80092004
+)
+
+// The functions bellow return an error even though they succeed.
```

Comment:
These functions seems inspired from mkcert. maybe it worth to put the link here in comment

Reply:
like this https://github.com/massalabs/station/pull/954/files#r1252752318 ?

Reply:
we will do it in a following PR